packages/gafl-webapp-service/src/routes/__tests__/error-routes.spec.js

```diff
@@ -155,6 +170,21 @@ describe('Error route handlers', () => {
         await serverError(request, mockToolkit)
         expect(consoleErrorSpy).toHaveBeenCalled()
       })
+
+      it.each([
+        [['kl', 'vu'], 'kl', 'vu'],
+        [['si', 'we'], 'we', 'si']
+      ])('sets altLang to be other item in two-item language array', async (locales, locale, altLang) => {
```

Comment:
I think it might be worth extracting this and the clientError into a separate describe block, given the only variation is calling `clientError` or `serverError`